### DISCUSSION TITLE
Provisioning Nginx - Apache

### DISCUSSION THREAD

**Discussion by santiagopereyramarchetti** [deployphp/deployer#3596] [General]
Hi, i love this repo, great job!!

Do you have any recipe for provisioning Nginx and Apache?

**Comment by antonmedv** [comment]
We have a little bit different provision recipe https://deployer.org/docs/7.x/getting-started#provision with Caddy.

**Comment by santiagopereyramarchetti** [comment]
I've already seen that.

But it would be great if the deployer had a recipe for nginx or apache, so the user can choose which  web server  to use. Do you have in mind to do something like this?

**Reply by antonmedv** [reply]
Maybe in the future.

**Reply by santiagopereyramarchetti** [reply]
I'm going to try to code for nginx!! if works well i will make a pull request.

Thkx.
